.circleci/config.yml

```diff
@@ -0,0 +1,48 @@
+# Java Gradle CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-java/ for more details
+#
+version: 2
+jobs:
+  test:
+    docker:
+    # specify the version you desire here
+    - image: circleci/openjdk:8-jdk
+
+    # Specify service dependencies here if necessary
+    # CircleCI maintains a library of pre-built images
+    # documented at https://circleci.com/docs/2.0/circleci-images/
+    # - image: circleci/postgres:9.4
+
+    working_directory: ~/repo
+
+    environment:
+      # Customize the JVM maximum heap limit
+      JVM_OPTS: -Xmx3200m
+      TERM: dumb

```

Comment:
Would be a good idea to add a `GRADLE_OPTS: --no-daemon` environment variable here. See [here](https://docs.gradle.org/current/userguide/build_environment.html#sec:gradle_environment_variables) for details on `GRADLE_OPTS`, and [here](https://docs.gradle.org/current/userguide/gradle_daemon.html) for info on the Gradle Daemon.

Reply:
From testing this setting GRADLE_OPTS seems to cause CircleCI to fail, but, if I just pass -no-daemon into ./gradlew it runs fine. Any ideas what could cause this?

Reply:
Think this should work now